composer.json

```diff
@@ -52,7 +53,7 @@
         "utopia-php/domains": "1.1.*",
         "utopia-php/swoole": "0.2.*",
         "utopia-php/storage": "0.5.*",
-        "utopia-php/websocket": "0.0.*",
+        "utopia-php/websocket": "dev-feat-onerror-callback",
```

Comment:
We have released a new tag with the new error handler: 0.1.0

Reply:
Resolved in next commit